res/css/views/rooms/_IRCLayout.pcss

```diff
@@ -19,48 +19,45 @@ $irc-line-height: $font-18px;
 .mx_IRCLayout {
     --name-width: 70px;
     --icon-width: 14px;
-    --right-padding: 5px;
+    --right-padding: 5px; /* TODO: Use a spacing variable */
 
     line-height: $irc-line-height !important;
 
-    .mx_EventTile {
-        --EventTile_irc_line-padding-block: 1px;
+    blockquote {
+        margin: 0;
+    }
 
-        /* timestamps are links which shouldn't be underlined */
-        > a {
-            text-decoration: none;
-            min-width: $MessageTimestamp_width;
-        }
+    .mx_EventTile[data-layout="irc"] {
```

Comment:
Protection against unintentional override by something in `.mx_EventTile:not([data-layout="bubble"])` on `_EventTile.pcss`